tst/JitterBufferFunctionalityTest.cpp

```diff
@@ -71,7 +71,7 @@ TEST_F(JitterBufferFunctionalityTest, continousPacketsComeInOrder)
     setPayloadToFree();
```

Comment:
Could you also add an inbound RTP stats API test in MetricsApiTest.cpp if it is possible to instrument a testing of the public API?